config/config_test.go

```diff
@@ -7,18 +7,13 @@ import (
 )
 
 func TestDefaultConfig(t *testing.T) {
-	expectedDefault := Config{
-		SegmentEvent:            "health",
-		ClusterID:               "",
-		DCOSVersion:             "",
-		DCOSClusterIDPath:       "/var/lib/dcos/cluster-id",
-		SignalServiceConfigPath: "/opt/mesosphere/etc/dcos-signal-config.json",
-		ExtraJSONConfigPath:     "/opt/mesosphere/etc/dcos-signal-extra.json",
-		DCOSVariant:             "UNSET",
+	d := DefaultConfig()
+	if d.SegmentEvent != "health" {
+		t.Error("Expected default segment event to be health, got", d.SegmentEvent)
```

Comment:
We can leave these here since `testing` lib logger acts like `.Println` and adds the spaces for you.